docker-init.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/sh
+CFGPATH="/dockershare"
+KEYPATH="/lumen/lumen.p12"
+die(){
+    echo "Exiting due to error: $@" && exit 1
+}
+do_config_fixup(){
+           sed -i -e "s,connection_info.*,connection_info = \"host=db port=5432 user=lumina password=1\"," \
+	   /lumen/config.toml
+}
+use_default_config(){
+    echo "No custom config.toml found, creating secure default."
+    tee /lumen/config.toml <<- EOF > /dev/null
+	[lumina]
+	bind_addr = "0.0.0.0:1234"
+	use_tls = true
+	server_name = "lumen"
+	[lumina.tls]
+	server_cert = "${KEYPATH}"
+	[database]
+	connection_info = "host=db port=5432 user=lumina password=1"
+	use_tls = false
+	[api_server]
+	bind_addr = "0.0.0.0:8082"
+	EOF
+}
+use_default_key(){
+    openssl req -x509 -newkey rsa:4096 -keyout /lumen/lumen_key.pem -out /lumen/lumen_crt.pem -days 365 -nodes \
```

Comment:
Starting IDA 7.5 (or maybe earlier?), IDA requires TLS certificates to have the "server authentication" extension.
openssl.conf should be modified before generating the certificate, [instructions here](https://github.com/naim94a/lumen/wiki/Creating-TLS-Certificates).

Reply:
I am using IDA 7.5 and didn't run into any issues with the generated key.  Either way, I added the attribute to `openssl.cnf` and added the `v3_req` extension to the `openssl` commandline when generating certificates.  If people bring their own certs, I don't think there's anything I can do to make sure it's formatted right.  They'd have to do that themselves.

Reply:
I guess IDA doesn't check on every OS... or maybe it depends on the sp?
either way, thanks for this PR, it's really great!